### PR TITLE
Improbadoc formatting for ecs docs (second half)

### PR DESCRIPTION
**Pull request by jessicafalk**
**Contributions**: We are not currently taking public contributions - see our [contributions](CONTRIBUTING.md) policy. However, we are accepting issues and we do want your [feedback](../README.md#give-us-feedback).

-------

#### Description
first half done by Firtina

#### Tests
How did you test these changes prior to submitting this pull request?
What automated tests are included in this PR?
#### Documentation
How is this documented (for example: release note, upgrade guide, feature page, in-code documentation)?
#### Primary reviewers
If your change will take a long time to review, you can name at most two primary reviewers who are ultimately responsible for reviewing this request. @ mention them.

### CODE REVIEW

File: docs/content/ecs/ecs-system-update-order.md [hunk 1]
```diff
@@ -30,12 +30,12 @@ The groups are executed in the following order:
...
-  * `CustomSpatialOSSendGroup` -  This group contains all systems running custom replication logic. If you create a [custom replication system](./custom-replication-system.md), add your system to this group.
-  * `InternalSpatialOSCleanGroup` - This group is used by the `CleanReactiveComponentsSystem` to remove all [reactive](./reactive-components.md) and [temporary components](./temporary-components.md) at the end of each update loop. **(Internal use only)**
+  * `CustomSpatialOSSendGroup` -  This group contains all systems running custom replication logic. If you create a [custom replication system]({{urlRoot}}/content/ecs/custom-replication-system.md), add your system to this group.
+  * `InternalSpatialOSCleanGroup` - This group is used by the `CleanReactiveComponentsSystem` to remove all [reactive]({{urlRoot}}/content/ecs/reactive-components.md) and [temporary components]({{urlRoot}}/content/ecs/temporary-components.md) at the end of each update loop. **(Internal use only)**
 
 Here's a diagram of the update order:
 
-![Update order](../../assets/update-order.png)
+![Update order]({{assetRoot}}assets/update-order.png)
```
Comment (JonasImprobable) [inline docs/content/ecs/ecs-system-update-order.md]: Could this URL be missing a "/"?

File: docs/content/ecs/ecs-component-generation.md [hunk 2]
```diff
@@ -2,17 +2,17 @@
...
 
 ## Overview
 
 A `struct`, which implements `Unity.Entities.IComponentData` and `Improbable.Gdk.Core.ISpatialComponentData`, is generated for each SpatialOS component. It has the same name as the corresponding schema component.
 The generation process creates a namespace for each struct according to the relevant schemalang component name: `{name of schema component}.Component`
-These structs only contain the defined schema data fields. They do *not* contain any fields or methods relating to [commands](./sending-receiving-component-commands.md) or [events (SpatialOS documentation)](./events.md) defined on that component.
+These structs only contain the defined schema data fields. They do *not* contain any fields or methods relating to [commands]({{urlRoot}}/content/ecs/sending-receiving-component-commands) or [events (SpatialOS documentation)](https://docs.improbable.io/reference/latest/shared/glossary#event) defined on that component.
 
 ## Generation details
 Each struct contains the following fields:
```
Comment (jamiebrynes7) [inline docs/content/ecs/ecs-component-generation.md]: List whitespace again

File: docs/content/ecs/ecs-system-update-order.md [hunk 3]
```diff
@@ -3,7 +3,7 @@
 [//]: # (TODO - use discussions about content in here https://docs.google.com/document/d/1MPTP1qEo9LaYxFGLQFEN2SqEzu9MxlKjVfOYKPUbTXg/edit)
 
 # (ECS) System update order
- _This document relates to the [ECS workflow](../intro-workflows-spos-entities.md)._
+ _This document relates to the [ECS workflow]({{urlRoot}}/content/intro-workflows-spos-entities.md)._
 
 Unity provides attributes to define the [update order of systems (Unity documentation)](https://github.com/Unity-Technologies/EntityComponentSystemSamples/blob/master/Documentation/content/ecs_in_detail.md#system-update-order). These attributes are: `UpdateInGroup`, `UpdateBefore` and `UpdateAfter`.
 > Note: You can only have one attribute of each type assigned to a system. If multiple are assigned they might override each other.
```
Comment (jamiebrynes7) [inline docs/content/ecs/ecs-system-update-order.md]: Whitespace required

File: docs/content/ecs/ecs-system-update-order.md (hunk 1 above)
Comment (jamiebrynes7) [reply docs/content/ecs/ecs-system-update-order.md]: It appears to render okay - I think `assetRoot` must have a trailing /

File: docs/content/ecs/ecs-system-update-order.md [hunk 4]
```diff
@@ -3,7 +3,7 @@
 [//]: # (TODO - use discussions about content in here https://docs.google.com/document/d/1MPTP1qEo9LaYxFGLQFEN2SqEzu9MxlKjVfOYKPUbTXg/edit)
 
 # (ECS) System update order
- _This document relates to the [ECS workflow](../intro-workflows-spos-entities.md)._
+ _This document relates to the [ECS workflow]({{urlRoot}}/content/intro-workflows-spos-entities.md)._
```
Comment (firtina-improbable) [inline docs/content/ecs/ecs-system-update-order.md]: `.md` extension (i see you removed it from other links)

File: docs/content/ecs/ecs-system-update-order.md (hunk 1 above)
Comment (samc-improbable) [reply docs/content/ecs/ecs-system-update-order.md]: AssetRoot requires `assets/` with no preceding /. So this is correct.

File: docs/content/ecs/ecs-system-update-order.md (hunk 3 above)
Comment (samc-improbable) [reply docs/content/ecs/ecs-system-update-order.md]: Done.

File: docs/content/ecs/ecs-component-generation.md (hunk 2 above)
Comment (samc-improbable) [reply docs/content/ecs/ecs-component-generation.md]: Done

**Pull request merged**